src/frontends/android/app/src/main/java/org/strongswan/android/data/VpnProfileDataSource.java

```diff
@@ -122,25 +142,36 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion)
 			if (oldVersion < 5)
 			{
 				db.execSQL("ALTER TABLE " + TABLE_VPNPROFILE + " ADD " + KEY_MTU +
-						   " INTEGER;");
+						" INTEGER;");
 			}
 			if (oldVersion < 6)
 			{
 				db.execSQL("ALTER TABLE " + TABLE_VPNPROFILE + " ADD " + KEY_PORT +
-						   " INTEGER;");
+						" INTEGER;");
 			}
 			if (oldVersion < 7)
 			{
 				db.execSQL("ALTER TABLE " + TABLE_VPNPROFILE + " ADD " + KEY_SPLIT_TUNNELING +
-						   " INTEGER;");
+						" INTEGER;");
 			}
 			if (oldVersion < 8)
 			{
 				db.execSQL("ALTER TABLE " + TABLE_VPNPROFILE + " ADD " + KEY_LOCAL_ID +
-						   " TEXT;");
+						" TEXT;");
 				db.execSQL("ALTER TABLE " + TABLE_VPNPROFILE + " ADD " + KEY_REMOTE_ID +
-						   " TEXT;");
+						" TEXT;");
 			}
+			//fancyfon
+			if (oldVersion < 5)
+			{
+				db.execSQL("ALTER TABLE " + TABLE_VPNPROFILE + " ADD " + KEY_ALLOWED_APPLICATIONS +
```

Comment:
I understand that they are fancyfon changes itd. But maybe put fancyfon and strongswan updates under the same if condition, and extract fancyfon  exec method to some  beautifully name method...